exist-core/src/main/java/org/exist/xquery/functions/fn/ExtCollection.java

```diff
@@ -60,8 +60,8 @@ public class ExtCollection extends Function {
                             " Documents contained in sub-collections are also included. If no value is supplied, the statically know documents are used, for the REST Server this could be the addressed collection.",
                     new SequenceType[]{
                             //Different from the official specs
-                            new FunctionParameterSequenceType("collection-uris", Type.STRING,
-                                    Cardinality.ZERO_OR_MORE, "The collection-URIs for which to include the documents")},
+                            new FunctionParameterSequenceType("collection-uri", Type.STRING,
```

Comment:
This change will likely break some code that is out in the wild.

Reply:
I guess you are right indeed.  I remember that I once used a construct like  `fn:collection(("/db/1","/db/2", "/db/3"))` but I also remember.... it was not that useable. A long time ago....

Reply:
Yes, this will need to be a major release item.

Reply:
Yes v7 would be appropriate.